bots/discord/nodemon.js/egg-discord-js-nodemon.json

```diff
@@ -0,0 +1,106 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-07-28T10:30:57+02:00",
+    "name": "discord.js nodemon",
+    "author": "nick@blackfortmc.com",
+    "description": "a nodemon discord js bot egg\r\n\r\nThis will clone a git repo for a bot. it defaults to master if no branch is specified.\r\n\r\nInstalls the node_modules on install. If you set user_upload then I assume you know what you are doing.",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:debian_nodejs-14",
+        "quay.io\/parkervcp\/pterodactyl-images:debian_nodejs-12",
+        "quay.io\/parkervcp\/pterodactyl-images:debian_nodejs-10"
+    ],
+    "file_denylist": [],
+    "startup": "if [[ -d .git ]] && [[ 0 == \"1\" ]]; then git pull; fi; if [[ ! -z ${NODE_PACKAGES} ]]; then \/usr\/local\/bin\/npm install ${NODE_PACKAGES}; fi; if [ -f \/home\/container\/package.json ]; then \/usr\/local\/bin\/npm install --production; fi; bash --rcfile <(echo 'export RUN_CMD=\"\/home\/container\/node_modules\/nodemon\/bin\/nodemon.js \/home\/container\/'{{BOT_JS_FILE}}' &\"; eval $(echo $RUN_CMD)')",
```

Comment:
There's no need to export inside the startup just for the other variable on the same line. It works exactly the same with

`/home/container/node_modules/nodemon/bin/nodemon.js /home/container/{{BOT_JS_FILE}}`

Reply:
There have been changes to the Discord.js generic egg, which you should integrate into this one, such as ability to uninstall packages